data/json/items/comestibles/frozen.json

```diff
@@ -0,0 +1,155 @@
+[
+  {
+    "type": "COMESTIBLE",
+    "id": "milkshake",
+    "name": "milkshake",
+    "name_plural": "milkshakes",
+    "//": "they bring the boys to the yard",
+    "description": "An all-natural cold beverage made with milk and sweeteners.  Tastes great when frozen.",
+    "weight": 270,
+    "volume": 1,
+    "container": "styrofoam_cup",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "color": "white",
+    "spoils_in": 24,
+    "nutrition": 40,
```

Comment:
That's way too much for a cup of milk. Ordinary milk has 8 nutrition, condensed sweetened milk has only 11.

Reply:
Milkshakes are made also with flavoring (such as chocolate, fruit, or candy), clean water, and sugar. I can bump down the nutrition, but I will bump up the quench (as clean water has 50 quench.)